apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -52,6 +54,22 @@ type AttestationTableType = {
   onChainId?: number | null;
 };
 
+const contributionToTableTypeMapper = (
+  contributions: UIContribution[],
+): AttestationTableType[] =>
+  contributions.map(c => ({
+    id: c.id,
+    date_of_submission: c.date_of_submission,
+    date_of_engagement: c.date_of_submission,
+    attestations: c.attestations,
+    guilds: c.guilds.map((guildObj: any) => guildObj.guild.name)[0] ?? '---',
```

Comment:
guiildObj should not be any we should be able to get the type from ui-types since it is on UIContribution